autogpt_platform/backend/.env.example

```diff
@@ -57,6 +57,7 @@ GOOGLE_CLIENT_SECRET=
 OPENAI_API_KEY=
 ANTHROPIC_API_KEY=
 GROQ_API_KEY=
+OPEN_ROUTER_API_KEY=
 
```

Comment:
is this one we plan to offer by default?

Reply:
Yep! I've shared the credentials with you in the usual way.